code/modules/hydroponics/fermenting_barrel.dm

```diff
@@ -75,9 +75,14 @@
 		icon_state = "barrel"
 	return ..()
 
-/datum/crafting_recipe/fermenting_barrel
-	name = "Wooden Barrel"
-	result = /obj/structure/fermenting_barrel
-	reqs = list(/obj/item/stack/sheet/mineral/wood = 8)
-	time = 50
-	category = CAT_PRIMAL
+/obj/structure/fermenting_barrel/distiller
+	name = "Distiller"
```

Comment:
To be pedantic, "distilling" refers to concentrating alcohol by controlled heating (the ethanol, with a lower boiling point than water, boils off, and is collected, creating a more concentrated drink). The stage involving a barrel, when the alcohol is originally created, is "fermenting".